apis/apis-overview/cloud-cost-api/cloud-cost-trends-api.md

```diff
@@ -0,0 +1,91 @@
+# Cloud Cost Trends API
+
+{% hint style="warning" %} This feature is only available in Kubecost Aggregator. For more information, see the release notes for [Kubecost V1.107](https://github.com/kubecost/cost-analyzer-helm-chart/releases/tag/v1.107.0). {% endhint %}
+
+{% swagger method="get" path="/cloudCost/view/trends" baseUrl="http://<your-kubecost-address>/model" summary="Trends API" %} {% swagger-description %} Analyzes change in cloud costs relative to a previous window of the same size {% endswagger-description %}
+
+{% swagger-parameter in="path" name="window" required="true" type="string" %}
+Duration of time over which to query. Compares cost usage of window to cost usage window of equal size directly preceding it. Accepts all standard Kubecost window formats (See our doc on using [the `window` parameter](https://docs.kubecost.com/apis/apis-overview/assets-api#using-window-parameter)).
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="names" required="true" type="string" %}
+Determines order sequence of queried items via comma-separated list. Dependent on the value of `aggregateBy` to list items. See more [below](cloud-cost-trends-api.md#using-the-names-parameter).
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="aggregate" type="string" required="false" %}
+Field by which to aggregate the results. Accepts: `invoiceEntityID`, `accountID`, `provider`, `service`, and `label:<name>`. Supports multi-aggregation using comma-separated lists, such as `aggregate=accountID,service`. Defaults to `service`.
```

Comment:
I'm realizing now that we actually support more than this and that the default is not actually `service` but an empty string. Using an empty string for aggregate would cause us to aggregate by item. 

As for a list of acceptable aggregate parameters, we can take `invoiceEntityID`, `accountID`, `provider`, `providerID`, `category`, `service`, and `label:<name>`

Reply:
So if they don't provide a value for aggregate, they will have to list every single line item in Names for a complete breakdown of their trends?

Reply:
Yes they would